syntax = "proto3";
package strategy;




import "model/easy_flow/deploy_strategy.proto";

/*
List请求
*/
message ListRequest {
    /*
    appId
    */
    string appId = 1 ;
}

/*
List返回
*/
message ListResponse {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    备注信息
    */
    string msg = 2 ;
    /*
    策略列表
    */
    repeated easy_flow.DeployStrategy data = 3 ;
}

/*
ListApi返回
*/
message ListResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    strategy.ListResponse data = 4 ;
}

